teps/0060-remote-resource-resolution.md

```diff
@@ -319,19 +324,252 @@ Why should this TEP _not_ be implemented?
 
 ## Alternatives
 
-<!--
-What other approaches did you consider and why did you rule them out?  These do
-not need to be as detailed as the proposal, but should include enough
-information to express the idea and why it was not acceptable.
--->
-
-## Infrastructure Needed (optional)
-
-<!--
-Use this section if you need things from the project/SIG.  Examples include a
-new subproject, repos requested, github details.  Listing these here allows a
-SIG to get the process for these resources started right away.
--->
+### Do Nothing
```

Comment:
I think it's good to have this one documented, but I would not go for this option!

Reply:
Yeah you're right - since our many conversations on this TEP and elsewhere I am starting to think about how / if we want to utilize this tekton-wide rather than only focus on the pipelines-specific use-cases.

Reply:
Updated to be "Do Nothing Specific to Tekton Pipelines"